platform/ios/app/MBXViewController.m

```diff
@@ -709,6 +724,79 @@ - (void)removeParkLayer
     [self.mapView.style removeLayer:parkLayer];
 }
 
+- (void)styleFilteredFill
+{
+    // set style and focus on Texas
+    [self.mapView setStyleURL:[NSURL URLWithString:@"mapbox://styles/justin/citc7ya7w00002jpa0z16cjah"]];
```

Comment:
This will only work with your access token, not the access token that the user enters. Instead, check in the style JSON as part of this project and refer to the bundled JSON file with a file URL or relative path.